doc/classes/@GlobalScope.xml

```diff
@@ -2823,6 +2823,7 @@
 			Hints that a [Color] property should be edited without affecting its transparency ([member Color.a] is not editable).
 		</constant>
 		<constant name="PROPERTY_HINT_OBJECT_ID" value="22" enum="PropertyHint">
+			Hints that the property's value is an object encoded as [ObjectID], with its type specified in the hint string. Used by the debugger.
```

Comment:
"object encoded as ObjectID" probably doesn't fully make sense, but it's related to `EncodedObjectAsID` class, so idk.